components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3252,6 +3245,13 @@ void RewardsServiceImpl::OnFetchBalance(FetchBalanceCallback callback,
                                         ledger::BalancePtr balance) {
   auto new_balance = std::make_unique<brave_rewards::Balance>();
 
+  LOG(INFO) << "BALANCE REPORT";
+  LOG(INFO) << "total: " << balance->total;
+  for (auto const& rate : balance->rates) {
```

Comment:
what was reasoning behind adding rates log here?

Reply:
verification

Reply:
didn't mean to commit this